pkg/apis/networkoperator/v1/networkconfig_types.go

```diff
@@ -110,6 +110,9 @@ type AdditionalNetworkDefinition struct {
 	// The name of the network. This will be populated in the resulting CRD
 	Name string `json:"name"`
 
+	// The resourceName of SR-IOV device. This will be populated in the resulting CRD
+	ResourceName string `json:"resourceName"`
```

Comment:
We need to document the AdditionalNetwork block in the README - could you please add a basic section that matches the other configuration variables?

Reply:
Just realize that we might not want to expose this ResourceName in current release.
Reason is that:
ResourceName specified in network CRD shall match with resourceName exposed by device plugin, but before device plugin is launched, there is no way for network-admin to know which resourceName shall be used. In current release, all devices found by SR-IOV device plugin will be exposed as the same resourceName (for example, openshift.com/sriov), so perhaps we could set the resourceName in the code when openshiftSRIOV network type is specified and not to bother network-admin to set a fixed value of resourceName.